src/ios/FirebasePluginMessageReceiver.h

```diff
@@ -1,5 +1,6 @@
-#import <Foundation/Foundation.h>
+#import "FirebasePluginMessageReceiver.h"
```

Comment:
This does not make sense - why would this file import itself?

Reply:
Thanks sir for your reply. I agree with you but I don't know why XCode suggest me this alternative and when I do it, it solves my bug automatically.